client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -255,6 +256,44 @@ export const useStocktakeColumns = ({
           }
         },
       },
+      {
```

Comment:
would you mind changing the `let` above to `const` the variable `total` isn't reassigned, so doesn't need to be a `let` (lines 223 & 242)

Reply:
[19b9b54](https://github.com/openmsupply/open-msupply/pull/1098/commits/19b9b547419b49259beb34ac375b1f3046d4cd96) all changes (: